Formula/kallisto.rb

```diff
@@ -0,0 +1,26 @@
+class Kallisto < Formula
+  desc "Quantify abundances of transcripts from RNA-Seq data"
+  homepage "https://pachterlab.github.io/kallisto/"
+  url "https://github.com/pachterlab/kallisto/archive/v0.43.1.tar.gz"
+  sha256 "2164938c2c61c04e338c4c132cf749f56d39e6f0b4c517121bca1fbc218e430e"
+
+  needs :cxx11
+  depends_on "cmake" => :build
+  depends_on "hdf5"
+
+  def install
+    ENV.cxx11
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+    doc.install "README.md"
```

Comment:
This is probably installed automatically, can you try and remove that line?

Reply:
This causes `README.md` to be placed in `Cellar/kallisto/0.43.1/share/doc/kallisto/` (instead of `Cellar/kallisto/0.43.1`) and symlinked from `share/doc/kallisto/`, do we not want that?

Reply:
<sup>bump</sup>

Reply:
Metafiles almost universally live in the prefix & aren't linked elsewhere.

Reply:
Good to know, thanks.